extensions-c9s.yaml

```diff
@@ -6,6 +6,10 @@ repos:
   - sig-virtualization
 
 extensions:
+  # https://issues.redhat.com/browse/RFE-4177
+  crun-wasm:
```

Comment:
should be wasm per update to https://github.com/openshift/machine-config-operator/pull/3776/files

Reply:
Fixed